code/src/components/Loader.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled, { keyframes } from 'styled-components/macro'
+
+export const Loader = () => {
+  const isLoading = useSelector((state) => state.ui.isLoading)
+
+  return (
+    <>
+      {isLoading &&
+        <LoaderContainer>
+          <Loading>
+            <EmojiLoad>🥜</EmojiLoad>
+            <EmojiLoad>🥚</EmojiLoad>
+            <EmojiLoad>🥛</EmojiLoad>
+          </Loading>
+        </LoaderContainer>
+      }
+    </>
+  );
+};
+
+const LoaderContainer = styled.div`
+  background-color:transparent;
+  display: flex;
+  margin: auto;
+  justify-content: cemter;
+`
+
+const Loading = styled.div`
+  margin: auto;
+  padding-top: 100px;
+`
+
+const bounce = keyframes` 
+  0%,
+  50%,
+  100% {
+    -webkit-transform: translateY(0);
+    -ms-transform: translateY(0);
+    -o-transform: translateY(0);
+    transform: translateY(0);
```

Comment:
Nice with vendor prefixes! You can actually skip defining them, though, as styled-components uses [stylis](https://github.com/thysultan/stylis.js) behind the scenes to automatically generate these vendor prefixes for you